app/models/teacher_training_adviser/steps/start_teacher_training.rb

```diff
@@ -5,6 +5,11 @@ class StartTeacherTraining < GITWizard::Step
     validates :initial_teacher_training_year_id, inclusion: { in: :year_ids }
 
     NOT_SURE_ID = 12_917
+    STUDYING_NOT_FINAL_YEAR = {
+      second_year: 222_750_002,
```

Comment:
Might be a good idea to have these constants in a Module that is included higher up in the hierarchy, or just included in the components that need them.